src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -25,12 +37,29 @@ export interface ConversationProps {
 
 export const PAGE_SIZE: number = 15
 
+const StyledBox = styled(Flex)`
+  border-top: 1px solid ${color("black10")};
+  padding: 15px 20px;
+  color: ${color("black60")};
+`
+
 const Conversation: React.FC<ConversationProps> = props => {
   const { conversation, relay, showDetails, setShowDetails } = props
+  const { user } = useSystemContext()
 
   const bottomOfPage = useRef(null)
   const initialMount = useRef(true)
 
+  const isMakeOfferArtwork =
+    // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
Was this code copied from elsewhere? If so, we shouldn't also be copying over these ts ignores

Reply:
It wasn't copied over `conversation.items[0]` was throwing a type error: `Object is possibly 'null'` then when I added `conversation.items?[0]?.item?.__typename` I was getting `Property 'item' does not exist on type 'number[]'.` for item.

Reply:
Ah 👍If having a type issue like this then will want to ping #dev-help or reach out! Seems like a quick fix.

Reply:
I was think that was an appropriate use of `// @ts-expect-error STRICT_NULL_CHECK` since most of the properties on the conversation object that can be nullable use it and it was added fairly recent as part of the strict null check work.

Reply:
If you have a second I'd be down to resolve it.

Reply:
So the history there is -- The `// @ts-expect-error STRICT_NULL_CHECK` flags were inserted by an automatic codemod script so that we could add strict type checking to Force without having to fix every error in the codebase first. With the codemod run, its understood that all new code should be type safe (and handle strict type errors like the above). 

(Strangely, CI should have been failed the build since it keeps track of the number of current flags in the codebase, and if any new ones are added should throw an error 🤔) 

Whats missing is a proper check after items: `conversation.items?.[0]...` (note the `?.` before array access).